packages/web-runtime/src/services/auth/authService.ts

```diff
@@ -350,6 +350,11 @@ export class AuthService implements AuthServiceInterface {
     this.authStore.clearUserContext()
   }
 
+  public async getRefreshToken() {
+    const user = await this.userManager.getUser()
+    return user.refresh_token
```

Comment:
Is it guaranteed that we have a user here? Or should we return `user?.refresh_token`

Reply:
Better be safe, yes!